### ISSUE TITLE
CreditCardView typescript error when getting element

### ISSUE THREAD

**Issue by Maldris** [triniwiz/nativescript-stripe#34]
### Which platform(s) does your issue occur on?

Android at local build time to device, both with CLI and sidekick

### Please, provide the following version numbers that your issue occurs with:

- CLI: 5.0.1
- Cross-platform modules: 4.2.0
- Runtime(s): 4.2.0
- Plugin(s): 
```
"dependencies": {
    "nativescript-background-http": "3.3.0",
    "nativescript-stripe": "5.0.1",
    "nativescript-theme-core": "1.0.4",
    "nativescript-ui-autocomplete": "3.9.0",
    "nativescript-ui-calendar": "3.8.1",
    "nativescript-ui-listview": "3.7.0",
    "nativescript-ui-sidedrawer": "4.3.0",
    "reflect-metadata": "0.1.12",
    "rxjs": "6.2.2",
    "tns-core-modules": "4.2.0",
    "zone.js": "0.8.26"
  },
  "devDependencies": {
    "babel-traverse": "6.26.0",
    "babel-types": "6.26.0",
    "babylon": "6.18.0",
    "lazy": "1.0.11",
    "nativescript-dev-typescript": "0.7.2",
    "nativescript-dev-webpack": "0.18.0",
    "tns-platform-declarations": "4.2.0",
    "typescript": "~2.7.2"
  }
```

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.

Attempting to setup a basic credit card input to generate a token as the README.md describes, I have: 

`<stripe:CreditCardView id="card"/>` 

with

```
const ccView: CreditCardView = page.getViewById("card") as CreditCardView;
    payments.getTokenFromView(ccView)
    .then((token) => { ... });
```

in the code behind file, where payments is a light module I used to wrap behaviour like getting tokens to make it convenient if I need to do so from multiple pages:

```
    public getToken(cc: Card): Promise<Token> {
        return new Promise((resolve, reject) => {
            this.stripe.createToken(cc, (error: Error, token: Token) => {
                if (!error) {
                    resolve(token);
                } else {
                    // TODO: handle error modal here
                    // console.log(error);
                    reject(error);
                }
              });
        });
    }

    public getTokenFromView(view: CreditCardView): Promise<Token> {
        return this.getToken(view.card);
    }
```

however when trying to build the project I get the tslint error that carries over to a build error in webpack

`TS2322: Type 'ViewBase' is not assignable to type 'CreditCardView'.`
from `const ccView: CreditCardView = page.getViewById("card");`

or 

`TS2352: Type 'ViewBase' cannot be converted to type 'CreditCardView'.` from `const ccView: CreditCardView = page.getViewById("card") as CreditCardView;`

tslint in VS code for the second of those suggests to convert to 'unknown' first, when trying to do that I get the error `TS2304: Cannot find name 'unknown'.`

I'm sure I've missed something simple in setup somewhere, but given I'm following what appears to be a simple example from the readme I wanted to ask here to help moth myself, and potentially others that encounter the issue in the future.

**Comment by RobertGardner** [comment]
Yes, it looks like there's a bug in the typings file, `index.d.ts`. I'll fix that. (See #35)

In the meantime, you can fix your code by converting to `any` first. (I'm not sure why it suggests converting to `unknown`.) Like this:

```
const ccView: CreditCardView = page.getViewById("card") as any as CreditCardView;
```

Thank you for reporting this.

**Comment by RobertGardner** [comment]
Fixed in 5.0.3.

**Comment by Maldris** [comment]
Thanks for the help, thought I'd missed something obvious but apparently not.
I'll update formally once npm catches up (still at 5.0.1), for now I'll manually import from github so I can keep working.

Thanks again.
